implementations/tutorial_multi_qpu_classification.py

```diff
@@ -0,0 +1,298 @@
+r"""
+.. _multiclass_margin_classifier:
+
+Multiclass margin classifier
+============================
+
+In this tutorial, we show how to use the PyTorch interface for PennyLane 
+to implement a multiclass variational classifier. We consider the iris database
+from UCI, which has 4 features and 3 classes. We use multiple one-vs-all
+classifiers with a margin loss (see `Multiclass Linear SVM
+<http://cs231n.github.io/linear-classify/>`__) to classify data. Each classifier is implemented
+on an individual variational circuit, whose architecture is inspired by
+`Farhi and Neven (2018) <https://arxiv.org/abs/1802.06002>`__ as well as
+`Schuld et al. (2018) <https://arxiv.org/abs/1804.00633>`__.
+
+|
+
+.. figure:: ../implementations/multi_qpu_classification/margin_2.png
+    :align: center
+    :width: 100%
```

Comment:
```suggestion
    :width: 50%
```
Just noticed it renders quite large at 100%
![image](https://user-images.githubusercontent.com/49409390/74277539-b9675480-4ce5-11ea-9c3c-c5b094b8e23d.png)